modules/table/src/main/java/org/apache/ignite/internal/table/distributed/storage/InternalTableImpl.java

```diff
@@ -446,14 +446,16 @@ private <R> CompletableFuture<R> enlistWithRetry(
                                         requestFunction.apply(tx.commitPartition(), primaryReplicaAndTerm.get2())
                                 );
                             } catch (PrimaryReplicaMissException e) {
-                                throw new TransactionException(e);
+                                throw new TransactionException(INTERNAL_ERR, e);
```

Comment:
It seems to me, that using `TransactionException` with the `INTERNAL_ERR` code is weird. Let's check and rework (if it is needed) our TX exception handling in a separate ticket.

Reply:
I did it just because the constructor with only Throwable as argument is deprecated. Let's remove the error code and rework this later.